components/annot-discrete-choices.vue

```diff
@@ -199,10 +211,12 @@
                         if ( this.relevantColumns.includes(columnName) ) {
 
                             if ( this.isMissingValue(columnName, transformedTable[index][columnName]) ) {
+
                                 // TODO: this string should be replaced by an app-wide way to designate missing values
                                 transformedTable[index][columnName] = this.missingValueLabel;
                             } else {
-                                transformedTable[index][columnName] = this.transformedValue(columnName, transformedTable[index][columnName]);
+
+                                transformedTable[index][columnName] = this.transformedValue(columnName, this.getOriginalColumnValue(transformedTable[index]["participant_id"], columnName));
```

Comment:
Yes, I like this solution! Excellent find, @surchs. 

In fact, the loop can remain at O(n) – or most accurately mn where _m_ represents the number of columns in `relevantColumns` – rather than the worst case O(n^2) if we convert the check `if ( this.relevantColumns.includes(columnName) )` into the second loop itself: `for column of this.relevantColumns`. (Of course, in this test case, we know _m_ is *1* since `relevantColumns` only includes `sex`).

And thus we can remove (for now) the need for this new store function.

I'll make the necessary changes.